src/main/java/de/zalando/aruha/nakadi/config/SecurityConfiguration.java

```diff
@@ -1,46 +1,75 @@
 package de.zalando.aruha.nakadi.config;
 
-import java.text.MessageFormat;
-
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
-
-import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
 import org.springframework.security.oauth2.config.annotation.web.configuration.ResourceServerConfigurerAdapter;
 import org.springframework.security.oauth2.config.annotation.web.configurers.ResourceServerSecurityConfigurer;
 import org.springframework.security.oauth2.provider.token.ResourceServerTokenServices;
 
-import org.zalando.stups.oauth2.spring.server.TokenInfoResourceServerTokenServices;
+import java.text.MessageFormat;
+
+import static org.springframework.http.HttpMethod.GET;
+import static org.springframework.http.HttpMethod.POST;
+import static org.springframework.http.HttpMethod.PUT;
 
 @EnableResourceServer
 @Configuration
 public class SecurityConfiguration extends ResourceServerConfigurerAdapter {
 
     private static final Logger LOG = LoggerFactory.getLogger(SecurityConfiguration.class);
-    public static final String UID = "uid";
 
-    @Value("${nakadi.oauth2.tokenInfoUri}")
-    private String tokenInfoUri;
+    @Autowired
+    private SecuritySettings settings;
+
+    @Autowired
+    private ResourceServerTokenServices tokenServices;
 
-    @Value("${nakadi.oauth2.clientId}")
-    private String clientId;
+    @Value("${nakadi.oauth2.scopes.uid}")
+    private String uidScope;
 
-    @Value("${nakadi.oauth2.enabled:true}")
-    private boolean oauthEnabled;
+    @Value("${nakadi.oauth2.scopes.nakadiRead}")
+    private String nakadiReadScope;
+
+    @Value("${nakadi.oauth2.scopes.eventTypeWrite}")
```

Comment:
@tmuehl but they should have other scope names because current ones will be owned by our team, so the other team will not be able to manage access to their nakadi instance.
So for me it seems to be quite a realistic scenario that another team will want to have an instance and will want to have their own scopes.